mappings/net/minecraft/world/World.mapping

```diff
@@ -80,30 +318,192 @@ CLASS net/minecraft/class_18 net/minecraft/world/World
 		ARG 6 sizeZ
 	METHOD method_218 isBoxSubmergedInFluid (Lnet/minecraft/class_25;)Z
 		ARG 1 box
+	METHOD method_219 loadEntitySkin (Lnet/minecraft/class_57;)V
+		ARG 1 entity
+	METHOD method_220 canMonsterSpawn ()Z
+	METHOD method_221 getRainGradient (F)F
+		ARG 1 delta
 	METHOD method_222 getTopY (II)I
-	METHOD method_223 (IIII)Z
+		ARG 1 x
+		ARG 2 z
+	METHOD method_223 setBlockMetaWithoutNotifyingNeighbors (IIII)Z
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
+		ARG 4 meta
+	METHOD method_224 playNoteBlockActionAt (IIIII)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 soundType
+		ARG 5 pitch
+	METHOD method_225 isFireOrLavaInBox (Lnet/minecraft/class_25;)Z
+		ARG 1 box
+	METHOD method_226 unloadEntitySkin (Lnet/minecraft/class_57;)V
+		ARG 1 entity
+	METHOD method_227 tickEntities ()V
+	METHOD method_228 getTopSolidBlockY (II)I
+		ARG 1 x
+		ARG 2 z
 	METHOD method_229 setBlock (IIII)Z
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
-		ARG 4 id
-	METHOD method_235 (IIII)V
+		ARG 4 blockId
+	METHOD method_230 worldEvent (IIIII)V
+		ARG 1 event
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+		ARG 5 data
+	METHOD method_231 remove (Lnet/minecraft/class_57;)V
+		ARG 1 entity
+	METHOD method_232 doLightingUpdates ()Z
+	METHOD method_233 getSpawnPositionValidityY (II)I
+		ARG 1 x
+		ARG 2 z
+	METHOD method_234 isAir (III)Z
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD method_235 blockUpdate (IIII)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 blockId
+	METHOD method_236 serverRemove (Lnet/minecraft/class_57;)V
+		ARG 1 entity
+	METHOD method_237 updateSkyBrightness ()V
+	METHOD method_238 hasChunk (II)Z
+		ARG 1 x
+		ARG 2 z
+	METHOD method_239 isPosLoaded (III)Z
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD method_240 setBlocksDirty (IIII)V
+		ARG 1 x
+		ARG 2 z
+		ARG 3 minY
+		ARG 4 maxY
+	METHOD method_241 updateEntity (Lnet/minecraft/class_57;)V
+		ARG 1 entity
+	METHOD method_242 tick ()V
+	METHOD method_243 blockUpdateEvent (III)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD method_244 notifyNeighbors (IIII)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 blockId
+	METHOD method_245 updateWeatherCycles ()V
+	METHOD method_246 setBlockDirty (III)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD method_247 camTransferPowerInDirection (IIII)Z
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 direction
+	METHOD method_248 manageChunkUpdatesAndEvents ()V
+	METHOD method_249 hasSkyLight (III)Z
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
-	METHOD method_243 (III)V
+	METHOD method_250 isEmittingRedstonePowerInDirection (IIII)Z
```

Comment:
[isEmittingRedstonePower](https://maven.fabricmc.net/docs/yarn-1.20.6+build.3/net/minecraft/world/RedstoneView.html#isEmittingRedstonePower(net.minecraft.util.math.BlockPos,net.minecraft.util.math.Direction)) in Yarn